### DISCUSSION TITLE
[지식공유]퍼블리싱의 순서

### DISCUSSION THREAD

**Discussion by Chominsu98** [PocketBand/frontend-web#12] [지식자료공유]
https://fromnowwon.tistory.com/entry/%EB%B0%98%EC%9D%91%ED%98%95-%EB%AA%A8%EB%B0%94%EC%9D%BC-%ED%8D%BC%EC%8A%A4%ED%8A%B8-css

@jaehawilly 
@Oneul1213 
- 우리가 하는 방식대로 요즘은 모바일 퍼스트 방식으로 퍼블하는게 대다수라고 해서 참고

**Comment by Oneul1213** [comment]
디자인 관련 참고할만한 내용들
1. [Material Design](https://m2.material.io/design/introduction)
2. [MUI](https://mui.com/material-ui/)

- 사실상 같은 내용이긴 한데, 1번은 디자인 방법론 이런 느낌이고, 2번은 프론트에서 안드로이드 MaterialUI랑 똑같이 쓰는 컴포넌트 doc

3. [MUI for Figma](https://www.figma.com/community/file/1062292530585891793)
- 피그마 라이브러리인 듯 한데, 공식판은 아니고 커뮤니티판인 듯 함. 그래도 열어보니 MUI 디자인 시스템 구현이 잘 되어 있어서, 이거 잘 갖다쓰면 좋을 듯 해서 참고용으로 같이 적어 둠
